packages/env/lib/env.js

```diff
@@ -219,6 +222,52 @@ module.exports = {
 	ValidationError,
 };
 
+/**
+ * Checks for legacy installs and provides
+ * the user the option to delete them.
+ *
+ * @param {Object} spinner A CLI spinner which indicates progress.
+ */
+async function checkForLegacyInstall( spinner ) {
+	const basename = path.basename( process.cwd() );
+	const installs = [
+		`../${ basename }-wordpress`,
+		`../${ basename }-tests-wordpress`,
+	];
+	await Promise.all(
+		installs.map( ( install ) =>
+			fs
+				.access( install )
```

Comment:
I wish `fs.exists` wasn't deprecated—it's so much more readable 😢

Reply:
Yeah